Dockerfile

```diff
@@ -1,21 +1,26 @@
 # Use the official Ubuntu 22.04 image as a base for the final image
-FROM ubuntu:22.04 as base
+FROM ubuntu:22.04 AS base
 
 # Install necessary packages for the final image
 RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
     curl sudo gpg lsb-release software-properties-common \
     && curl -fsSL https://deb.nodesource.com/setup_lts.x | bash - \
     && apt-get install -y apt-utils \
     && apt-get install -y nodejs \
-    && npm install -g npm@latest
+    && npm install -g npm@latest \
+    && apt-get update && apt-get install -y git
 
 # Create the 'masa' user and set up the home directory
 RUN useradd -m -s /bin/bash masa && mkdir -p /home/masa/.masa && chown -R masa:masa /home/masa
 
+
 # Switch to user 'masa' for following commands
 USER masa
 WORKDIR /home/masa
 
+# add VERSION file for the build stage
+COPY --chown=masa:masa internal/constants/version.go ./internal/constants/version.go
```

Comment:
just a Q - nothing blocking here. Why we need this? isn't this covered already by L32 here berlow?

Reply:
I'll check but previously it wasn't available in the run context.